platform/linux-generic/odp_packet_io.c

```diff
@@ -377,7 +377,7 @@ static const char *driver_name(odp_pktio_t hdl)
 
 	entry = get_pktio_entry(hdl);
 	if (entry == NULL) {
-		ODP_ERR("pktio entry %d does not exist\n", hdl);
+		ODP_ERR("pktio entry %" PRIuPTR " does not exist\n", (uintptr_t)hdl);
```

Comment:
Would it be better to use odp_pktio_to_u64() to get a printable value? OTOH, this is odp_packet_io.c which presumably knows very well what a pktio handle really is.

(And the same comment to other similar changes, e.g. in odp_timer.c)